src/libsync/owncloudpropagator.h

```diff
@@ -571,6 +572,7 @@ private slots:
     AccountPtr _account;
     QScopedPointer<PropagateDirectory> _rootJob;
     SyncOptions _syncOptions;
+    int m_jobScheduled;
```

Comment:
minor: would prefer = 0; here instead of in ctor
minor: we use only `_`, not `m_` in OCC